packages/components/src/internal/components/entities/EntityCrossProjectSelectionConfirmModal.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FC, memo } from 'react';
+
+import { ConfirmModal } from '../base/ConfirmModal';
+import { capitalizeFirstChar } from '../../util/utils';
+
+import { getCrossFolderSelectionMsg } from './utils';
+
+interface Props {
+    crossFolderSelectionCount: number;
+    currentFolderSelectionCount: number;
+    noun?: string;
+    nounPlural?: string;
+    onDismiss: () => void;
+    title?: string;
+    verb?: string;
+}
+
+export const EntityCrossProjectSelectionConfirmModal: FC<Props> = memo(props => {
+    const {
+        onDismiss,
+        verb = 'Edit ',
```

Comment:
I probably wouldn't include the space in the verb since it's hard for the user to know that is required.